zammad_py/api.py

```diff
@@ -288,6 +294,21 @@ def download(self, id, article_id, ticket_id):
         return self._raise_or_return_json(response)
 
 
+class TicketArticlePlain(Resource):
+
+    path_attribute = 'ticket_article_plain'
+
+    def download(self, article_id):
```

Comment:
`download` function is redundant here. We can use [this](https://github.com/joeirimpan/zammad_py/blob/develop/zammad_py/api.py#L210) instead.

```python
client.ticket_article_plain.find(1)
```